README.md

```diff
@@ -40,9 +39,152 @@ These resultSync API methods all return Promises:
 | resultSync method | Description |
 | -------------------- | ----------- |
 | `resultSync.manager.list` | list all results |
-| `resultSync.manager.create(workflow)` | create a result |
-| `resultSync.manager.read(workflowId)` | read a result |
-| `resultSync.manager.update(workflow)` | update a result |
+| `resultSync.manager.create(result)` | create a result |
+| `resultSync.manager.read(resultId)` | read a result |
+| `resultSync.manager.update(result)` | update a result |
+
+### Topic Subscriptions
+
+#### wfm:result:create
+
+##### Description
+
+Creating a new Result
+
+##### Example
+
+
+```javascript
+var parameters = {
+  resultToCreate: {
+    //A Valid JSON Object
+  },
+  //Optional topic unique identifier.
+  topicUid: "uniquetopicid"
+}
+
+mediator.publish("wfm:result:create", parameters);
+```
+
+#### wfm:result:read
+
+##### Description
+
+Read a single Result
+
+##### Example
+
+
+```javascript
+var parameters = {
+  id: "resultId",
+  //Optional topic unique identifier.
+  topicUid: "uniquetopicid"
+}
+
+mediator.publish("wfm:result:read", parameters);
+```
+
+#### wfm:result:update
+
+##### Description
+
+Update a single Result
+
+##### Example
+
+
+```javascript
+var parameters = {
+  resultToUpdate: {
+    ...
+    id: "resultId"
+    ...
+  },
+  //Optional topic unique identifier.
+  topicUid: "uniquetopicid"
+}
+
+mediator.publish("wfm:result:update", parameters);
+```
+
+
+#### wfm:result:remove
+
+##### Description
+
+Remove a single Result
+
+##### Example
+
+
+```javascript
+var parameters = {
+  id: "resultId",
+  //Optional topic unique identifier.
+  topicUid: "uniquetopicid"
+}
+
+mediator.publish("wfm:result:remove", parameters);
+```
+
+
+#### wfm:result:list
+
+##### Description
+
+List All Results
+
+##### Example
+
+
+```javascript
+var parameters = {
+  //Optional topic unique identifier.
+  topicUid: "uniquetopicid"
+}
+
+mediator.publish("wfm:result:list", parameters);
+```
+
+
+### Published Topics
+
+The following topics are published by this module. Developers are free to implement these topics subscribers, or use a module that already has these subscribers implement (E.g. the [raincatcher-sync](https://github.com/feedhenry-raincatcher/raincatcher-sync) module).
+
+
+| Topic         | Description           |
```

Comment:
I'm not sure about that. I think the module API (which in effect the topics are) should be documented in the module itself.

I think we should reference an example implementation of the implementation of these topics (the raincatcher-sync module) but the module should not be bound to all of them.

So a user installing the module standalone should not have to go looking for module documentation in another module.